### ISSUE TITLE
The code sucks……

### ISSUE THREAD

**Issue by cuancuancuanhao** [namedysx/CRAFT-tensorflow#3]
There were too many problems when I managed to move the model onto TensorRT.

The code using the switch 'is_training == True' all the time both training or testing, which made 'normalizer_fn' and 'normalizer_params' worked quietly in every convolution layer.
So, I had to create self-normalization layer in the new network, and attached them to everywhere like 'conv -> BatchNorm -> selfNorm -> ReLU' or 'conv -> selfNorm -> BatchNorm -> ReLU' (BTW, 'is_training' was written incorrectly as 'is_trianing' in the file).

There was no explicit operation on 'normalizer_fn' and 'normalizer_params' recommended in Tensor-flow,  so part of the noted parameters  was invalid, such as all moving mean are 0 and all moving variance are 1. So I had to create the transformer matrix just between before and after the corresponding layers (Thanks to the transformation was linear ones).

...

**Comment by devaroundu** [comment]
Hi  cuancuancuanhao ,
Could you please share your changes ?
